### PR TITLE
Bug 1815901 - Further document experimentation for cookie banner handling

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

**Review by MozillaNoah — APPROVED** [review]

File: nimbus.fml.yaml [hunk 1]
```diff
@@ -215,13 +215,20 @@ types:
       description: The identifiers for the sections of the MR 2022.
       variants:
         feature-ui:
-          description: Indicates if the user interfaces should be visible.
+          description: An integer either 0 or 1 indicating if the UI for cookie banner handling should be visible,
+          0 to hide the UI and 1 to show the UI. The actual UI is composed by cookie banner section 
+          in the settings page, the toolbar section and the re-engagement dialog.
```
Comment (gabrielluong) [inline nimbus.fml.yaml:220]: I think the failure is complaining because of missing indents here.

Comment (Amejia481) [reply nimbus.fml.yaml:220]: Thanks!

File: nimbus.fml.yaml [hunk 2]
```diff
@@ -215,13 +215,20 @@ types:
...
         feature-setting-value:
-          description: Indicates if the cookie handling setting should be enabled.
+          description: An integer either 0 or 1 indicating if cookie banner setting should be enabled or disabled,
+            0 for setting the value to disabled, 1  for enabling the setting with the value reject_all.
         dialog-re-engage-time:
-          description: Indicates after how many hours of interaction, the dialog should show again.
+          description: An integer indicating the number of hours that needs to happen before
+          the re-engagement dialog shows again since the last seen, for example if set to 4 
+          that means if the users has seen the dialog, it will see it 4 hours later.
```
Comment (gabrielluong) [inline nimbus.fml.yaml:227]: This probably also needs an indent?

Comment (Amejia481) [reply nimbus.fml.yaml]: I re-push adding indentation thanks 🙏🏽 !

**Review by gabrielluong — APPROVED** [review]

**Pull request merged**